sdk-cpp/src/internal/download_interface.h

```diff
@@ -3,9 +3,9 @@
 
 #include "do_download_status.h"
 
-#if defined(DO_INTERFACE_COM)
 #include "do_download_property.h"
 
+#if defined(DO_INTERFACE_COM)
```

Comment:
>DO_INTERFACE_COM [](http://example.com/codeflow?start=12&length=16)

The SetCallback method is no longer under ifdef, so this one shouldn't be either, no?

Reply:
Right, will remove